ComplexProperties/AttributedString.cs

```diff
@@ -0,0 +1,137 @@
+// ---------------------------------------------------------------------------
```

Comment:
Possibly in a separate checkin, but we should revise all the headers to reflect the MIT license.  Ping me offline if you want to agree on the content before you take the time to apply to all files.